configs/bundle_config.yaml

```diff
@@ -159,6 +159,14 @@ databundles:
     output:
     - data/natura.tiff
 
+  bundle_cutout_sea:
+    countries: [ASEAN]
+    category: cutouts
+    destination: "cutouts"
+    urls:
+      zenodo:
+      gdrive:
```

Comment:
Nice! The preparation is good.
The missing parts are:
- there is the need to add the flag "output: [Files]" to specify the output files that are expected
- there is the need to fill the url of at least gdrive.

Regarding the second point, if you share the gdrive link of the zipped file, then, we can load into our account and then provide the appropriate link.
Be aware that the zip file shall be created using [this script](https://github.com/pypsa-meets-earth/pypsa-earth/blob/main/scripts/non_workflow/zip_folder.py)
You should be able to do that using: `zipFilesInDir("./cutouts", "cutouts.zip", lambda x: True, include_parent=False)`

As a comment, it may be interesting to discuss together on Thursday the name of the cutout file

Reply:
Just added cutout output as you suggested, thanks! Regarding a link to the cutout you can find it here -> https://drive.google.com/file/d/1qg_FQvB81gxh6xAm0lrZJ3KJyespHYb_/view?usp=share_link